src/inputs/Textarea.vue

```diff
@@ -4,11 +4,13 @@
         <textarea
             v-on:input="e => this.onChange(e.target.value)"
             :value="inputValue"
+            :maxlength="maxlength"
             :id="this.id"
             :rows="this.height"
             :name="this.name"
             :readonly="this.readonly">
         </textarea>
+        <span>{{ this.ui.maxlength && getTextCount }}</span>
```

Comment:
this could potentially throw `cannot read property 'maxlength' of undefined`
You've made a check for this already below (lines 45-48): why not use that instead?